app/models/plugins/call_tool.rb

```diff
@@ -57,15 +58,37 @@ def json_targets
 
   # Returns [{ code: <country-code>, name: <country-name>}, {..} ...]
   def target_countries
-    locale = page.language&.code || :en
     targets.map(&:country_code).uniq.map do |country_code|
+      country = ISO3166::Country[country_code]
       {
-        name: ISO3166::Country[country_code].translation(locale),
-        code: country_code
+        name: country.translation(page.language_code),
+        code: country_code,
+        phoneCode: "+#{country.country_code}"
       }
     end
   end
 
+  # Temporary ugly solution until we figure out if we want all countries here or
+  # only a subset.
+  def countries_phone_codes
+    list = ISO3166::Country.all.reject do |country|
+      country.country_code.blank?
+    end
+
+    list.map! do |country|
+      {
+        name: country.translation(page.language_code),
+        code: "+#{country.country_code}"
+      }
+    end
+
+    # Prioritize US
+    us = list.find { |c| c[:name] == ISO3166::Country['US'].translation(page.language_code) }
+    list.delete(us)
+    list.unshift(us)
+    list
```

Comment:
Didn't know about rotate, but I checked and it doesn't excactly do the same thing that I'm doing.